src/main/java/seedu/address/model/person/UniquePersonList.java

```diff
@@ -3,10 +3,7 @@
 import static java.util.Objects.requireNonNull;
 import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 
-import java.util.HashMap;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Need to edit this part :>

Reply:
Edited! Sorry for that, I think it is caused by an automatic formatting of IntelliJ.